aiaccel/torch/lightning/callback.py

```diff
@@ -0,0 +1,27 @@
+import json
+
+import lightning.pytorch as pl
+
+
+class SaveParamCallback(pl.Callback):
+    """
+    Lightning Callback for save paramater in fit ends.
+
+    Args:
+        param_name (str): Parameter name to save
+        output_path (str): File name to save
+    """
+
+    def __init__(self, param_name: str, output_path: str) -> None:
+        super().__init__()
+        self.param_name = param_name
+        self.output_path = output_path
+
+    def on_fit_end(self, trainer: pl.Trainer, pl_module: pl.LightningModule) -> None:
+        param_value = trainer.callback_metrics.get(self.param_name)
+        if param_value is not None:
+            param_value_item = param_value.item()
+            with open(self.output_path, "w") as f:
+                json.dump(param_value_item, f)
+        else:
+            print(f"Warning: '{self.param_name}' not found in callback_metrics.")
```

Comment:
これはエラーな気がします．
```suggestion
            raise KeyError('{self.param_name}' not found in callback_metrics.")
```
ただ，これだと，わざわざ自分でraiseしなくても，getの部分を[self.param_name]すれば同じようなエラー出ますね…

Reply:
[self.param_name] にして、文章を簡易化しました
c01d0f51570a4a7985299ac377f32ca9a9007747